tests/test_ism.py

```diff
@@ -8,7 +8,7 @@ def test_ism_enabled(ism_enabled_file, ism_disabled_file):
     with FileReader(ism_disabled_file) as reader:
         df_disabled = reader.to_pandas()
         assert not reader.idle_sleep_mode_activated
-    assert (df_enabled.X.groupby(level=0).count() == 30).all()
+    assert (df_enabled.X.groupby(level=0).count() == 1).all()
```

Comment:
Why did it change?

Reply:
Now we calculate sub-second timestamps so each index is unique.